### PR TITLE
Meenu/fix: faq search subtask

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
FAQ search subtasks

1. Fixed placeholder font-color to `typography-disabled`

2. Updated `font weight `to bold for "Results for"

3. Fixed padding and gap issues for search results and no search result

4. Added `break-word` for search input 

5. Fixed appearing results after clearing search input 

6. Added meta tag under `document.tsx `as input field was auto focusing on mobile devices.

### CODE REVIEW

File: test-script.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  return snakeCaseVariables;
+}
+
+function scanDirectory(directory) {
+  const files = fs.readdirSync(directory);
+
+  files.forEach((file) => {
+    const filePath = path.join(directory, file);
+    const fileStats = fs.statSync(filePath);
```
Comment (prince-deriv) [inline test-script.js:35]: This file is accidentally added.

**Review by prince-deriv — CHANGES_REQUESTED** [review]

**Review by prince-deriv — DISMISSED** [review]

File: apps/deriv-hk/pages/_document.tsx [hunk 2]
```diff
@@ -4,6 +4,11 @@ export default function Document() {
   return (
     <Html lang="en" className="bg-background-primary-container">
       <Head>
+        <meta
+          name="viewport"
+          key="viewport"
```
Comment (mohsen-deriv) [inline apps/deriv-hk/pages/_document.tsx:9]: we already have this in `_app.tsx`, do we need it here as well? 
it will create a warning on dev start.

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: libs/templates/src/lib/help-center/sections/search/search-results.tsx [hunk 3]
```diff
@@ -36,31 +36,36 @@ export const SearchResults = ({ onSearchKeyChange }: SearchResultsProps) => {
...
+                      );
+                    }),
+                  )}
+                </div>
               )}
+
               {!results.length && searchKey && (
                 <div className="flex w-full justify-center py-general-2xl">
-                  <Text size="lg">
+                  <Text size="lg" className="break-all text-typography-default">
```
Comment (mohsen-deriv) [inline libs/templates/src/lib/help-center/sections/search/search-results.tsx:68]: no need for `text-typography-default`, please remove it, thanks 🙏

Comment (mohsen-deriv) [reply libs/templates/src/lib/help-center/sections/search/search-results.tsx:68]: please update your branch with main and run `npm ci`

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/search-chip/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  inputClassName?: string;
+  size?: sizeVariant;
+};
+
+const sizeVariantClass: { [key in sizeVariant]: string } = {
+  sm: 'px-general-sm py-general-xs',
+  md: 'p-general-md',
+};
+
+export const SearchChip = ({
```
Comment (mohsen-deriv) [inline libs/components/src/lib/search-chip/index.tsx:22]: can we use `SearchInput` from quill-design please?

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/search-chip/index.tsx (hunk 4 above)
Comment (meenakshi-deriv) [reply libs/components/src/lib/search-chip/index.tsx]: removed this file as we are no more using search chip

**Pull request merged**